docs/stdlib/array.rst

```diff
@@ -197,7 +194,9 @@ Reference
 
 .. eql:operator:: arrayplus: array<anytype> ++ array<anytype> -> array<anytype>
 
-    Array concatenation.
+    Concatenates given arrays of :eql:type:`anytype` into one.
```

Comment:
```suggestion
    Concatenates two arrays of the same type into one.
```

Important to note that the types must be the same.

Reply:
I think we'll need to mirror this in a few more places.. Good catch!